assets/js/utils/rtcUtils.ts

```diff
@@ -24,7 +24,7 @@ export const mergedScreenRef: MergedScreenRef = {
   refreshId: undefined,
 };
 
-const sourceIds: { audio: string; video: string } = { audio: "", video: "" };
+const sourceIds: { audio: string; video: string; [key: string]: string } = { audio: "", video: "" };
```

Comment:
Why do we need this change?

Reply:
Unfortunately, the `MediaStreamTrack.kind`, used later in the file as a key to the object, is of type `string` even tho [it can only take values `video` or `audio`](https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack/kind), thus this change.